func/demo/va_arg.cpp

```diff
@@ -0,0 +1,83 @@
+#include <cstdarg>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <faasm/compare.h>
+#include <vector>
+
```

Comment:
Var-args isn't something we have modified or implemented, but is worth testing just to be sure it works (as it looked like this was the problem for a while).